test/lib/usd/translators/testUsdExportImportRoundtripPreviewSurface.py

```diff
@@ -188,7 +188,8 @@ def __testUsdPreviewSurfaceRoundtrip(self,
         imported_path = cmds.getAttr(file_node+".fileTextureName")
         # imported path will be absolute:
         self.assertFalse(imported_path.startswith(".."))
-        self.assertEqual(imported_path.lower(), original_path.lower())
+        self.assertEqual(os.path.normpath(imported_path.lower()),
+                         os.path.normpath(original_path.lower()))
```

Comment:
On some Windows configurations, normpath can be used to make sure both paths use the same directory separators.